src/strategies/eoa-balance-and-staking-pools/index.ts

```diff
@@ -0,0 +1,72 @@
+import { formatEther } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'DEFI-Foundation';
+export const version = '0.1.1';
+
+const tokenAbi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+const abiStakingPool = [
+  'function getSmartPoolVotes (address account, uint256 blockNumber) public view returns (uint256 votes)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag =
+    typeof snapshot === 'number'
+      ? snapshot
+      : await provider.getBlockNumber(snapshot);
```

Comment:
I have talked to my colleague and he said that better to use `latest`
```suggestion
      : 'latest';
```
because there is no need to make a request

Reply:
Hi, unfortunately using 'latest' as fallback it's not ideal for our specific case.
As you can see from the below screenshot the call request expect it to be a number and test fails.

![IMG_20231005_112812_963](https://github.com/snapshot-labs/snapshot-strategies/assets/106986644/ad8f1964-7c11-408c-b366-d22dc395d7cb)